pom.xml

```diff
@@ -557,6 +570,71 @@
 				</plugins>
 			</build>
 		</profile>
+		
+		<!-- Uses tycho-surefire instead of maven-surefire to run tests that require an Equinox platform -->
```

Comment:
This documentation seems to be inappropriate
```suggestion
		<!-- Uses maven-surefire instead of tycho-surefire to run tests that do not require an Equinox platform -->
```

Reply:
Done.